launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -35,13 +37,14 @@
     <arg name="image_number" value="$(var image_number)"/>
   </include> -->
 
-  <!-- Pointcloud map filter -->
+  <!-- detection preprocess module -->
```

Comment:
I prefer the name is `Pointcloud map filter`.

Reply:
Fixed at [499266a](https://github.com/autowarefoundation/autoware.universe/pull/961/commits/499266a96b5bb4ae0be85a796c2846484fa37ee4)